synopsys-task/src/synopsys-task/input.ts

```diff
@@ -187,6 +182,40 @@ export const POLARIS_BRANCH_PARENT_NAME = getInput(
   constants.POLARIS_BRANCH_PARENT_NAME_KEY_CLASSIC_EDITOR,
   null
 );
+
+export const POLARIS_ASSESSMENT_MODE =
+  taskLib.getInput(constants.POLARIS_ASSESSMENT_MODE_KEY)?.trim() ||
+  (taskLib
+    .getInput(constants.POLARIS_ASSESSMENT_MODE_KEY_CLASSIC_EDITOR)
+    ?.trim() === POLARIS_ASSESSMENT_MODES.CI
+    ? POLARIS_ASSESSMENT_MODES.CI
+    : taskLib
+        .getInput(constants.POLARIS_ASSESSMENT_MODE_KEY_CLASSIC_EDITOR)
+        ?.trim() === POLARIS_ASSESSMENT_MODES.SOURCEUPLOAD
+    ? POLARIS_ASSESSMENT_MODES.SOURCE_UPLOAD
+    : "");
```

Comment:
This can be extracted to a `private` method ex. `getPolarisAssessmentModeInput()` or `getInputForPolarisAssessmentMode()` and you can use that method here.

Reply:
I think we can just check if there is any value in POLARIS_ASSESSMENT_MODE_KEY_CLASSIC_EDITOR we can continue. No need to check CI or source upload. Bridge team will be doing that as well for sure. Please check once with team as well on this.

Reply:
Task.json
```
options": {
        "": "",
        "CI": "CI",
        "SOURCEUPLOAD": "SOURCE_UPLOAD"
      },
```
      @spurohitsynopsys we can not set this value (SOURCE_UPLOAD) as key, because I checked that _ is not supported in the key. That's why  we are using that mapping thing.

Reply:
No, i mean to say we can check if POLARIS_ASSESSMENT_MODE_KEY_CLASSIC_EDITOR is empty then we don't need to check anything i.e. CI, source upload. Whatever is coming bridge can handle that.
Ex: taskLib.getInput(constants.POLARIS_ASSESSMENT_MODE_KEY)?.trim()  || taskLib
        .getInput(constants.POLARIS_ASSESSMENT_MODE_KEY_CLASSIC_EDITOR)
        ?.trim() || ""
        
        why do we need to check each option?